### PR TITLE
Make pp visible if using nightly

### PR DESCRIPTION
**Pull request by BinAccel**
Plz don't break stuff.

### CODE REVIEW

File: templates/user/user-base.jade [hunk 1]
```diff
@@ -49,7 +49,7 @@ block body
                 b {% trans "Total points:" %}
                 |  #{user.points}
                 br
-                .pp-feature(style="display: none")
+                .pp-feature(style=(None if perms.judge.test_site else 'display: none'))
```
Comment (quantum5) [inline templates/user/user-base.jade]: I do not think that's valid. These are dictionary lookups, and when PyJade handles them, you are to write `perms['judge']['test_site']`.

**Comment by BinAccel** [comment]
@quantum5 fixed

File: templates/user/user-base.jade [hunk 2]
```diff
@@ -49,7 +49,7 @@ block body
                 b {% trans "Total points:" %}
                 |  #{user.points}
                 br
-                .pp-feature(style=(None if perms.judge.test_site else 'display: none'))
+                .pp-feature(style=(None if perms['judge']['test_site else'] 'display: none'))
```
Comment (quantum5) [inline templates/user/user-base.jade]: I do not know why `else` is in the string.

**Comment by BinAccel** [comment]
Take 3.

**Pull request merged**